external-crates/move/crates/move-package/tests/test_sources/resolve_pkg_name_dep_conflicting_dep_reg_overrides/Move.resolved

```diff
@@ -1,3 +1 @@
-Conflicting "regular" and "dev" overrides found in Root-rename:
-C-rename = { local = "deps_only/C-v1" }
-C-rename = { local = "deps_only/C-v2" }
+Can't have dev-dependency package and regular package with same name 'C-resolved'
```

Comment:
Well these should say `C-rename` but anyway this PR was just an attempt, I'm not sure the stopgap I was trying here checks out.